pyproject.toml

```diff
@@ -86,7 +86,7 @@ source = ["aind_data_access_api", "tests"]
 [tool.coverage.report]
 exclude_lines = [
     "if __name__ == .__main__.:",
-    "from",
+    "from .* import .*",
```

Comment:
I think you can add an anchor "^" so that only lines that begin with "from" and "import" will be ignored.